content/components/dialog.mdx

```diff
@@ -0,0 +1,383 @@
+
+### Side sheets
+
+Side sheets slide from the right or left edges of the viewport. **Left side sheets are reserved for global navigation drawers. Right side sheets are used for global actions, but can also be considered for quick previews in full-width pages, or configuration panels.**
+
+Side sheets take the entire height of the viewport, and should be used sparingly. Don't use side sheets to present create/edit forms, or flows that may contain a lot of information. For that, use a page instead.
+
+<img
+  width="960"
+  alt="placement-desktop-1"
+  src="https://user-images.githubusercontent.com/18661030/185214096-8ad908b4-2cbd-467d-8e74-bad6948c5a2d.png"
+/>
+<Caption>Left aligned side sheet is reserved for global navigation only.</Caption>
+
+<img
+  width="960"
+  alt="placement-desktop-2"
+  src="https://user-images.githubusercontent.com/18661030/185214094-419d2271-dbb9-4e78-90e6-4433ed9c42fb.png"
+/>
+<Caption>
+  Right aligned side sheet can be used for global actions, but may also be considered for quick previews or
+  configuration panels.
+</Caption>
+
+## Narrow-viewport placement
+
+### Bottom sheets
+
+A bottom sheet is a variant supported on narrow viewports made to facilitate reachability. Dialogs that appear centered in large screens may adapt to appear as a bottom sheet on small screens.
+
+A bottom sheet always dims the rest of the screen for focus and takes up the full width of the viewport. Tapping on the backdrop dismisses the Dialog.
+
+<img
+  width="960"
+  alt="bottom-sheet"
+  src="https://user-images.githubusercontent.com/18661030/185214084-43e9573a-1a3b-4b82-b121-bd03897f48ac.png"
+/>
+<Caption>Bottom sheet in portrait mode.</Caption>
+
+In landscape mode, a bottom sheet has a maximum width of `480px`, and is centered horizontally.
+
+<img
+  width="960"
+  alt="overlay-sheet-landscape"
+  src="https://user-images.githubusercontent.com/18661030/185214089-a1bb1a70-914b-4a8f-b395-9c279a88a1fe.png"
+/>
+<Caption>Bottom sheet in landscape mode.</Caption>
+
+### Full-screen
+
+Full-screen Dialogs may be used to present content that needs all the available space on narrow viewports. Dialogs that have one or more input fields should always use the full-screen variant.
+
+<img
+  width="960"
+  alt=""
+  src="https://user-images.githubusercontent.com/18661030/185214079-d30342c1-6061-4a31-ab65-c0cef0f00278.png"
+/>
+<Caption>Full-screen dialog in portrait mode.</Caption>
+
+## Accessibility
+
+Dialog's title and close button as addressed previously are required to ensure an accessible experience for everyone. Further, keyboard and focus behavior must be considered.
+
+When a Dialog is opened, the first interactive element (the close button) is focused. Hitting `tab` will cycle through all interactive elements within the Dialog. To escape the focus trap, hitting the `esc` key or clicking on the backdrop will close the Dialog. While the Dialog is open, page scrolling is disabled and becomes `inert`.
```

Comment:
Does the focus behavior change when using a search field in the `Subheader` region or a Command palette in the `Header` region?

<img width="505" alt="Screenshot 2022-08-25 at 11 35 27" src="https://user-images.githubusercontent.com/912236/186631087-664c4836-3429-4cf5-841f-1d63952746f5.png">

Reply:
Great question! Yes, since the input comes before the close button in the dom it should receive focus first. I added this to the accessibility section!